index.bs

```diff
@@ -1064,6 +1067,12 @@ associated.
     ::  This member describes the [=[RPS]=]' requirements regarding availability of the [=Client-side-resident Credential 
         Private Key=]. If the parameter is set to true, the authenticator MUST create a 
         [=Client-side-resident Credential Private Key=] when creating a [=public key credential=].
+
+    :   <dfn>requireUserVerification</dfn>
+    ::  If this parameter is set to <i>true</i>, only authenticators capable of user verification would participate in the 
+        {{CredentialsContainer/create()}} operation, thereby ensuring the authenticator will perform the user verification
```

Comment:
I disagree with @jyasskin here. The behavior he describes wrt user verification is how things work today and everyone understands that. however, after grovelling through UAF, U2F, and WebAuthn, AFAICT we do not explicitly state it. I submited #493.